sway/input/cursor.c

```diff
@@ -174,11 +174,15 @@ void cursor_rebase_all(void) {
 	}
 }
 
-static int hide_notify(void *data) {
-	struct sway_cursor *cursor = data;
+void cursor_hide(struct sway_cursor *cursor) {
```

Comment:
nit: can you make this static since it isn't listed in the header (and shouldn't unless it is ever used in a different file)?

Reply:
Done